orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/config/PreconfiguredJobStageProperties.java

```diff
@@ -24,7 +24,7 @@
 
 @Data
 
-public class PreconfiguredJobStageProperties {
+public abstract class PreconfiguredJobStageProperties {
```

Comment:
If you're going to be using `@Data`, you shouldn't use public properties, since the accessors will be generated.